draft-ietf-quic-recovery.md

```diff
@@ -937,13 +956,13 @@ loss_detection_timer:
 pto_count:
 : The number of times a PTO has been sent without receiving an ack.
 
-time_of_last_sent_ack_eliciting_packet:
+time_of_last_sent_ack_eliciting_packet\[kPacketNumberSpace]:
 : The time the most recent ack-eliciting packet was sent.
 
 largest_acked_packet\[kPacketNumberSpace]:
 : The largest packet number acknowledged in the packet number space so far.
 
-loss_time\[kPacketNumberSpace]:
+loss_times\[kPacketNumberSpace]:
```

Comment:
Revert this change. We use singular for other arrays, which reads better to me anyways.